ipv4_test.go

```diff
@@ -74,3 +75,33 @@ readFrame:
 		}
 	}
 }
+
+func TestCloseUnblockPendingRead(t *testing.T) {
```

Comment:
Since it's now in unblocking mode, I feel like this test is less effective, as `ifce.Read` would return immediately regardless. But I guess we can leave this here in case we find a way to unblock blocking mode read too.